README.md

```diff
@@ -223,37 +230,35 @@ You can toggle the presenter or overview mode by pressing respectively `alt+p` a
 
 ## Fullscreen
 
-Fullscreen can be toggled via browser options, or by **hovering over the bottom right corner of your window until the fullscreen icon appears and clicking it**.
+Fullscreen can be toggled via browser options, or by hovering over the bottom right corner of your window until the fullscreen icon appears and clicking it.
 
 <a name="pdf-export"></a>
 
 ## PDF Export
 
-Exporting a totally sweet looking PDF from your totally sweet looking Spectacle presentation is absurdly easy. You can either do this via the browser, or from the command line:
+You can export a PDF from your Spectacle presentation either from the command line or browser:
 
 #### CLI
 
 - Run `npm install spectacle-renderer -g`
 - Run `npm start` on your project and wait for it to build and be available
 - Run `spectacle-renderer`
-- A totally cool PDF is created in your project directory
 
-For more options and configuration of this tool, check out:
+A PDF is created in your project directory. For more options and configuration of this tool, check out:
 
 [https://github.com/FormidableLabs/spectacle-renderer](https://github.com/FormidableLabs/spectacle-renderer)
 
 #### Browser
 
-- Run `npm start`
-- Open [http://localhost:3000/](http://localhost:3000/)
-- Add `export&` after the `?` on the URL of page you are redirected to, e.g.: [http://localhost:3000/#/?export&\_k=wbyhif](http://localhost:3000/#/?export&_k=wbyhif)
+After running `npm start` and opening [http://localhost:3000/#/](http://localhost:3000/#/) in your browser...
+
+- Add `?export&_k=wbyhif` after the `/` on the URL of page you are redirected to, e.g.: [http://localhost:3000/#/?export&\_k=wbyhif](http://localhost:3000/#/?export&_k=wbyhif)
```

Comment:
What does this `\_k=wbyhif` slug represent? We could probably use some clarification for the docs if we want to recommend using it

Reply:
Updating this thread per our discussion: we don't need it after all. 🙂